src/doc/stdmetadata.rst

```diff
@@ -288,10 +288,98 @@ The following metadata items are specific to photos or captured images.
 
     The exposure time (in seconds) of the captured image.
 
+.. option:: "Iso" : float
+
+    The ISO speed of the captured image.
+
+.. option:: "TStop" : float
+
+    The stop of the camera when it captured the image.
+
 .. option:: "FNumber" : float
 
     The f/stop of the camera when it captured the image.
 
+.. option:: "WhiteBalanceTemperature" : float
+
+    Color temperature in Kelvin of the camera when it captured the image.
+
+.. option:: "WhiteBalanceTint" : float
+
+    Green/magenta tint of the camera when it captured the image.
+
+.. option:: "FocalLengthNominal" : float
+
+    Mumber printed on barrel of a prime lens, or number next to index mark on a zoom lens, in units of millimeters.
+
+.. option:: "FocalLengthPinhole" : float
+
+    In the simplest model of image formation, the distance between the pinhole and the image plane, in units of millimeters.
+
+.. option:: "FocalLengthEffective" : float
+
+    In the thick lens model, the effective focal length is the distance between the front focal point and the front nodal point, or equivalently the back focal point and the back nodal point, in units of millimeters.
+
+.. option:: "SerialNumber" : string
+
+    For captured or scanned image, the Serial Number of the camera or scanner.
+
+.. option:: "FirmwareVersion" : string
+
+    For captured or scanned image, the Firmware Version of the camera or scanner.
+
+.. option:: "EntrancePupil" : float
+
+    The axial distance from the image plane to the entrance pupil, in units of millimeters.
+
+.. option:: "LensMake" : string
+
+    Manufacturer or vendor of the lens.
+
+.. option:: "LensModel" : string
+
+    Model name or model number of the lens.
+
+.. option:: "LensSerialNumber" : string
+
+    Serial number of the lens.
+
+.. option:: "LensFirmwareVersion" : string
+
+    Firmware version of the lens.
+
+.. option:: "PhotositeSensor" : float
+
+    Distance between centers of sensor photosites, in microns.
+
+.. option:: "Uuid" : string
+
+    Identifies this camera uniquely among all cameras from all vendors.
+
+.. option:: "Label" : string
+
+    Text label identifying how the camera was used or assigned.
+
+
+
+GPS
+===
+
+The following GPS-related metadata items are specific.
+
+.. option:: "Longitude" : float
+
+    The degrees, minutes, and seconds of longitude.
+
```

Comment:
No, no, there already is a "GPS Exif data" section in this document. It already describes longitude, latitude, and altitude. Unfortunately, they have different names -- AND TYPES! We should definitely not have two different "OIIO standard" names for the same thing.

It's totally valid to propose "let's switch to the OpenEXR convention and auto-translate when we read and write Exif", but you have never proposed that, nor it is not even what this PR is implementing.